packages/components/badge/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@consolelabs/badge",
+  "version": "0.0.1-rc7",
```

Comment:
Not sure the version can work with Changeset once we integrate the tool. Maybe change the version of all components to `0.0.1` for a safe integration?